CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ set(ARB_MODCC "" CACHE STRING "path to external modcc NMODL compiler")
 
 # Use libunwind to generate stack traces on errors?
 
-option(ARB_UNWIND "Use libunwind for stack trace printing if available" OFF)
+option(ARB_BACKTRACE "Use stack trace printing if available" OFF)
```

Comment:
Maybe change to "Use stack trace printing (requires Boost)"?
It isn't "if available", because `find_package` sets `REQUIRED` for Boost.